### DISCUSSION TITLE
what if youtube create a ai

### DISCUSSION THREAD

**Discussion by Ybouali** [community/community#84941] [Other Feature Feedback, Questions, & Ideas] [answered]
in this ai the user ask about a video ,
i know it is already there but what if that ai speaks directly with the user like ,
when user ask about charlie chaplin video's the ai will speak with the user and tell the user that ur friend  X from facebook have been seen this video 2 time and other stuff 
this ai will help friend know each other a lot what they like what is the common between them ,
of course it will be a problem of data between this too platform  
a don't know but it just a idea

**Comment by samus-aran** [comment, marked as answer]
Hi @Ybouali ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by Kanthasuwamy-Kumaran** [reply]
hello

**Reply by Aifranklin** [reply]
Hi

**Comment by Kanthasuwamy-Kumaran** [comment]
how to work

**Reply by Aifranklin** [reply]
Good

**Comment by Aifranklin** [comment]
YouTube
